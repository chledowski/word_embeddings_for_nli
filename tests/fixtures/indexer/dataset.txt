six seven, eight nine, ten
seven, ten, six
six unknown, unkunk nine, someword
11, seven seven, x