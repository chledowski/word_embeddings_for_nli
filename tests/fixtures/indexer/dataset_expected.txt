5 6 7 2, 5 8 9 2, ten
5 7 2, 5 10 2, six
5 6 1 2, 5 1 9 2, someword
5 1 2, 5 7 7 2, x